Fails on FreeBSD: error[E0433]: failed to resolve: use of undeclared crate or module `libc`

```
$ cargo build --release
...
   Compiling clap_derive v4.4.7
   Compiling parse_duration v2.1.1
error[E0433]: failed to resolve: use of undeclared crate or module `libc`
  --> /home/yonas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/network-interface-1.1.1/src/target/getifaddrs.rs:25:18
   |
25 |         unsafe { libc::freeifaddrs(self.base) }
   |                  ^^^^ use of undeclared crate or module `libc`

error[E0433]: failed to resolve: use of undeclared crate or module `libc`
  --> /home/yonas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/network-interface-1.1.1/src/target/getifaddrs.rs:31:20
   |
31 |     match unsafe { libc::getifaddrs(addr.as_mut_ptr()) } {
   |                    ^^^^ use of undeclared crate or module `libc`

error[E0433]: failed to resolve: use of undeclared crate or module `libc`
 --> /home/yonas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/network-interface-1.1.1/src/target/getifaddrs.rs:5:16
  |
5 |     base: *mut libc::ifaddrs,
  |                ^^^^ use of undeclared crate or module `libc`

error[E0433]: failed to resolve: use of undeclared crate or module `libc`
 --> /home/yonas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/network-interface-1.1.1/src/target/getifaddrs.rs:6:16
  |
6 |     next: *mut libc::ifaddrs,
  |                ^^^^ use of undeclared crate or module `libc`

error[E0433]: failed to resolve: use of undeclared crate or module `libc`
  --> /home/yonas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/network-interface-1.1.1/src/target/getifaddrs.rs:10:17
   |
10 |     type Item = libc::ifaddrs;
   |                 ^^^^ use of undeclared crate or module `libc`

error[E0433]: failed to resolve: use of undeclared crate or module `libc`
  --> /home/yonas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/network-interface-1.1.1/src/target/getifaddrs.rs:30:44
   |
30 |     let mut addr = mem::MaybeUninit::<*mut libc::ifaddrs>::uninit();
   |                                            ^^^^ use of undeclared crate or module `libc`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `network-interface` (lib) due to 6 previous errors
warning: build failed, waiting for other jobs to finish...
```

OS: FreeBSD 14.0-RELEASE
rustc 1.76.0-nightly

--- a/crates/upnp/Cargo.toml
+++ b/crates/upnp/Cargo.toml
@@ -21,7 +21,7 @@ tokio = {version = "1", features = ["macros"]}
 futures = "0.3"
 url = "2"
 async-recursion = "1"
-network-interface = "1"
+network-interface = { git = 'https://github.com/ikatson/network-interface', branch = "compile-on-freebsd" }
 
 [dev-dependencies]
 tokio = {version = "1", features = ["macros", "rt-multi-thread"]}

